src/models/AuthenticatedIdentity.ts

```diff
@@ -25,13 +22,21 @@ export class AuthenticatedIdentity {
    */
   public policy: string;
 
+  /**
+   * The COSE body of the authenticated identity.
+   */
+  public coseBody: string | undefined;

```

Comment:
For me this, this property changes the character of this class; it no longer encapsulates just an (authenticated) caller to the app, but the call itself. Consider renaming this class or refactoring such that `coseBody` and the authenticated identity becomes properties of another class.